android-exoplayer/src/main/java/com/brentvatne/exoplayer/ReactExoplayerView.java

```diff
@@ -296,6 +297,27 @@ public void onClick(View v) {
             }
         });
 
+        //Handling the playButton click event
+        ImageButton playButton = playerControlView.findViewById(R.id.exo_play);
```

Comment:
were these buttons not working before?

Reply:
@benoitdion - It's working before, but it's handled by the player control view by default. But now, I have handled the actions of play and pause of my own to update the "isPause" property.

Reply:
does this means you're overriding a built-in click listener with the one below or are the player control managing that differently?

Reply:
Yes... I am overriding a built-in click listener...But with the same default logic with additional handling of "ispaused" property.

`isPaused = paused;
if (player != null) {
            if (!paused) {
                startPlayback();
            } else {
                pausePlayback();
            }
        }`


In default, it will call `startPlayback()` and `pausePlayback()` on play and pause correspondingly